src/databricksbundle/_config/config.yaml

```diff
@@ -42,3 +44,9 @@ services:
   databricksbundle.storage.adapter.AzureGen2Configurator:
     tags:
       - 'databricks.storage.configurator'
+
+  databricksbundle.notebook.daipehelp.DaipeHelp:
+    arguments:
+      - 'https://github.com/daipe-ai/'
```

Comment:
either make this a parameter in the config or a constant in the `DaipeHelp` file